va/va.h

```diff
@@ -1021,6 +1021,13 @@ typedef enum {
      * VAConfigAttribValEncAV1Ext2 union.
      */
     VAConfigAttribEncAV1Ext2            = 54,
+    /** \brief delta QP per block attribute for Encoding.  Read-only.
+     *
+     * This attribute describes whether to support delta qp per block
+     * and the supported size of delta qp block. The value
+     * returned uses the VAConfigAttribValEncDeltaQpPerBlock type.
+     */
+    VAConfigAttribEncDeltaQpPerBlock    = 55,
```

Comment:
suggest to change to , perMBControl for future usage.

Reply:
Rename it to VAConfigAttribPerMBControl?